docs/ide-integration.md

```diff
@@ -3,11 +3,20 @@
 IDEs work better with shims than they do environment variable modifications. The simplest way is
 to add the rtx shim directory to PATH.
 
-For IntelliJ and VSCode—and likely others, you can modify `~/.zprofile`
-with the following:
+For IntelliJ and VSCode—and likely others, you can modify `~/.zprofile` if your are using zsh or `~/.bashrc` if you are using bash with the following:
```

Comment:
This isn't correct. 'bashrc' is only used for interactive sessions and "zprofile" is probably used unless chsh was used to modify the shell—and possibly not even then